src/Dsn.php

```diff
@@ -211,23 +217,17 @@ protected function getBaseEndpointUrl(): string
             $url .= $this->path;
         }
 
-        $url .= '/api/' . $this->projectId;
+        $url .= '/' . $this->projectId;
 
         return $url;
     }
 
     /**
-     * @see https://www.php.net/manual/en/language.oop5.magic.php#object.tostring
+     * Returns the base url to Sentry from the DSN.
      */
-    public function __toString(): string
+    private function getBaseEndpointUrl(): string
```

Comment:
This change is not documented

Reply:
I just moved the functions around, there is no change to document

Reply:
You made it private, but it's final (I just noticed), so it doesn't count.